hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2413,6 +2413,12 @@ ce = \case
     -- [ Default, Object, Tag ]
     -- [ False, True, Cond ]
     op "select"
+  DLE_ContractFromAddress _at _addr -> do
+    case maybeT T_Address of
+      T_Data maybeMap -> do
+        let la = DLLA_Data maybeMap "None" $ DLA_Literal DLL_Null
+        cla la
+      _ -> impossible "maybe not maybe"
```

Comment:
This is `cla $ mdaToMaybeLA T_Address Nothing`

https://github.com/reach-sh/reach-lang/blob/master/hs/src/Reach/AST/DLBase.hs#L467-L468

Reply:
What does `c` stand for in all of these functions?